src/components/Home/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { NavLink } from 'react-router-dom';
+
+import './index.css';
+
+
+
+class Home extends Component {
+    constructor (props){
+        super(props);
+        this.state = {
+            isClicked : false,
+            working: false
+        }
+    }
+
+
+
+handleClick(){
+    this.setState({
+       isClicked : !this.state.isClicked 
+    })
+}
+
+  render() {
+    return (
+    <div className = "homee"> 
+
+    <header>  
+             <h1 className = "HomeTitle"> WHAT CAN I SUGGEST YOU TONIGHT?</h1>
+       
+    </header>
+
+
+    <section className = "homeBouton">
+
+        <div className={this.state.isClicked ? 'Open_page' : 'page'}>
```

Comment:
On vas regarder ça ensemble

Reply:
je t'attends :)

Reply:
handleClick = () => {
    this.setState({
       isClicked : !this.state.isClicked 
    })
}